ci/playbooks/content_provider/run.yml

```diff
@@ -3,6 +3,8 @@
   gather_facts: true
   tasks:
     - name: Deploy content provider
+      environment:
+        ANSIBLE_CONFIG: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/ci-framework/ansible.cfg"
```

Comment:
hmmm - either the one provided by the framework, or the one actually used by the zuul job. IIRC they have a different log file. Might be good to use zuul's instead?

Let's see how it works with that one as a first iteration